app/Services/EDocument/Imports/ZugferdEDocument.php

```diff
@@ -65,11 +69,12 @@ public function run(): Expense
             $expense->currency_id = Currency::whereCode($invoiceCurrency)->first()->id;
```

Comment:
We should always fall back to the company currency in cases where the currency is not resolved:

company->settings->currency_id

Reply:
done. both in mindee and in zugferd implementations